CMakeLists.txt

```diff
@@ -14,11 +14,20 @@ ez_detect_project_name(DETECTED_NAME)
 
 set (EZ_SOLUTION_NAME ${DETECTED_NAME} CACHE STRING "The name of the solution")
 
-project (${EZ_SOLUTION_NAME}
-	VERSION 1.0
-	DESCRIPTION "An open-source game engine"
-	HOMEPAGE_URL "http://ezEngine.net"
-	LANGUAGES C CXX)
+if (CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
+	# C# support only on windows desktop.
+	project (${EZ_SOLUTION_NAME}
+		VERSION 1.0
+		DESCRIPTION "An open-source game engine"
+		HOMEPAGE_URL "http://ezEngine.net"
+		LANGUAGES C CXX CSharp)

```

Comment:
You could probably put the languages into a variable.

Reply:
Done!